sbin/auto_rainbow_grades.py

```diff
@@ -0,0 +1,119 @@
+"""
+Automatically generate rainbow grades.
+
+usage: python3 auto_rainbow_grades.py <semester> <course>
+
+If the instructor chooses to supply a custom_customization.json instead of using
+the web form this file will always take precedence and overwrite any customization.json
+generated by the web ui.
+"""
+
+# Imports
+import sys
+import os
+import shutil
+import pwd
+
+# Constants
+RAINBOW_GRADES_PATH = '/usr/local/submitty/GIT_CHECKOUT/RainbowGrades'
```

Comment:
paths /usr/local/submitty, /var/local/submitty, and user submitty_php should be read from config files
/usr/local/submitty/config/submitty.json 
/usr/local/submitty/config/submitty_users.json

(and you can get to those config files from ../.. etc from the relative location of the installed script)